meinberlin/apps/votes/dashboard.py

```diff
@@ -38,6 +38,7 @@ class GenerateVotesComponent(DashboardComponent):
     identifier = 'voting_token_generation'
     weight = 49
     label = _('Generate voting codes')
+    for_superuser = True
```

Comment:
think the wording is fine and easy to understand, imho it would be a tiny little bit more accurate:
`for_superuser_only`, because `for_superuser = False` sounds like all other users are can, except of superusers.